posts/dialog.md

```diff
@@ -0,0 +1,12 @@
+feature: dialog
+status: use
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of (HTML5.1)(http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
+  
+In its modal mode, it blocks all access to content behind the front-most dialog.
```

Comment:
Shouldn't this sentence be in the previous paragraph? In that paragraph we give a quick intro to the `dialog` element, in this one we're talking about the polyfill.